.github/workflows/tag.yml

```diff
@@ -37,13 +37,17 @@ jobs:
       - name: Build
         run: npm run ${{ steps.variables.outputs.BUILD_SCRIPT }} -- --env release=true --env basename="v/${{ steps.variables.outputs.VERSION }}" --env semver=${{ steps.variables.outputs.VERSION }} --env info_version=${{ steps.variables.outputs.VERSION }} --env github_actions=true
 
+      - name: Create latest version reference file
+        run: |
+          echo "{ \"latestVersion\" : \"${{ steps.variables.outputs.VERSION }}\" }" > ./dist/latestVersion.json
+
       - name: Azure Login
         uses: azure/login@v1.4.6
         with:
           creds: ${{ secrets.AZURE_CREDENTIALS }}
 
       - name: Azure Deploy
-        uses: azure/CLI@v1.0.6
+        uses: azure/CLI@v1
         with:
           inlineScript: |
             az storage blob upload-batch -s dist -d \$web --account-name ${{ steps.variables.outputs.AZURE_ACCOUNT_PROD }} --overwrite
```

Comment:
I'm seeing in `main.yml` we added `azcliversion: 2.43.0` in Azure Deploy section, but we seemingly forgot to add it for `tag.yml` ?

Reply:
Hm, don't know what that thing does.. But I guess we can add it 👍 😄